pkg/gpu/containers/containers.go

```diff
@@ -30,6 +31,7 @@ var numberedResourceRegex = regexp.MustCompile(`^nvidia([0-9]+)$`)
 
 const (
 	nvidiaVisibleDevicesEnvVar = "NVIDIA_VISIBLE_DEVICES"
+	dockerInspectTimeout       = 100 * time.Millisecond
```

Comment:
do we want to allow configuration here for quick remediations in case something goes south with this logic?